src/pagination/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from "./pagination-model.class";
 export * from "./pagination.component";
 export * from "./pagination.module";
+export * from "./pagination-nav/pagination-overflow.component";
```

Comment:
Try exporting everything by name here ... I'm guessing the compiler doesn't like the autogenerated names when it's trying to compile the root module.